symengine/lib/symengine_wrapper.pyx

```diff
@@ -2801,6 +2798,45 @@ cdef class Lambdify(object):
                 out = tmp
         return out
 
+cdef class LambdaDouble(_Lambdify):
+
+    cdef vector[symengine.LambdaRealDoubleVisitor] lambda_double
+    cdef vector[symengine.LambdaComplexDoubleVisitor] lambda_double_complex
+
+    cdef _init(self, symengine.vec_basic& args_, symengine.vec_basic& outs_):
+        if self.real:
+            self.lambda_double.resize(1)
+            self.lambda_double[0].init(args_, outs_)
+        else:
+            self.lambda_double_complex.resize(1)
+            self.lambda_double_complex[0].init(args_, outs_)
+
+    cdef void _eval_real(self, double[::1] inp, double[::1] out):
+        self.lambda_double[0].call(&out[0], &inp[0])
+
+    cdef void _eval_complex(self, double complex[::1] inp, double complex[::1] out):
+        self.lambda_double_complex[0].call(&out[0], &inp[0])
+
+
+IF HAVE_SYMENGINE_LLVM:
+    cdef class LLVMDouble(_Lambdify):
+
+        cdef vector[symengine.LLVMDoubleVisitor] lambda_double
+
+        cdef _init(self, symengine.vec_basic& args_, symengine.vec_basic& outs_):
+            self.lambda_double.resize(1)
+            self.lambda_double[0].init(args_, outs_)
+
+        cdef void _eval_real(self, double[::1] inp, double[::1] out):
+            self.lambda_double[0].call(&out[0], &inp[0])
+
+
+def Lambdify(args, exprs, bool real=True, bool llvm=False):
+    IF HAVE_SYMENGINE_LLVM:
+        if llvm:
+            return LLVMDouble(args, exprs, real)
```

Comment:
I agree, at least a warning?

We could also leave it `None` and allow the user to choose default by setting an environment variable, e.g. `SYMENGINE_LAMBDIFY_BACKEND` or something along those lines? (we could then run the tests suite with
that variable set to `llvm` and `lambda` respectively)